packages/zowe-explorer-api/src/vscode/ui/TableView.ts

```diff
@@ -56,7 +56,7 @@ export namespace Table {
     export type Callback = SingleRowCallback | MultiRowCallback | CellCallback;
 
     /** Conditional callback function - whether an action or option should be rendered. */
-    export type Conditional = (data: RowData | ContentTypes) => boolean;
+    export type Conditional = (data: RowData[] | RowData | ContentTypes) => boolean;
```

Comment:
Will this require a small changelog entry? 😋

Reply:
Yes, sorry I forgot to add - addressed in b07a19a